### PR TITLE
dispenser testing / tx composition refactoring

### PR DESCRIPTION
**Pull request by mattmarcello**

### CODE REVIEW

File: lib/domain/entities/compose_dispenser.dart [hunk 1]
```diff
@@ -1,16 +1,45 @@
-class ComposeDispenser {
+import "./compose_response.dart";
+import "./compose_fn.dart";
```
Comment (dime-squanderer) [inline lib/domain/entities/compose_dispenser.dart:2]: minor: is there a difference between import "package:path/to/file" vs using the path directly?

File: lib/presentation/screens/compose_dispenser/view/usecase.dart [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (dime-squanderer) [inline lib/presentation/screens/compose_dispenser/view/usecase.dart:1]: still needed?

File: integration_test/compose_dispenser_test.dart [hunk 3]
```diff
@@ -0,0 +1,701 @@
...
+      await tester.pumpAndSettle();
+
+      expect(
+          find.text(
+              'escrow quantity must be greater than or equal to give quantity'),
+          findsOneWidget);
+    });
+  });
+
+  group("Tx Submission", () {
```
Comment (dime-squanderer) [inline integration_test/compose_dispenser_test.dart:504]: should this test be re-added?

**Review by dime-squanderer — APPROVED** [review]

File: lib/domain/entities/compose_dispenser.dart (hunk 1 above)
Comment (mattmarcello) [reply lib/domain/entities/compose_dispenser.dart:2]: just me being lazy :)

**Pull request merged**